docs/requirements.txt

```diff
@@ -1,30 +1,30 @@
-# Direct dependencies
-mkdocs==1.4.2
```

Comment:
Those changes seem unrelated to the PR.

Reply:
Hm, imo they are related. 

My reasoning behind:

- to have version in the documentation I use macro addon
- I install a new dependency (that implies I need to update the requirements.txt) to have that addon
- I created a new requirements.txt using `pip freeze` from my local python virtual environment to get a snapshot of dependency tree to avoid manually changing the requirements.txt

I decided to go the easier way and updated requirements.txt in one run, but if those changes are really unrelated I can split it into 2 or 3 PRs, like one pr with new dependency, one pr with new formatting through `pip freeze` (because it kinda makes sense to maintain files automatically instead of manually building them, saves from making mistakes), one PR with actual changes in mkdocs.toml and may be even one more for the changes in markdown files, but the question is, do we really want that level of granularity of actions?

Reply:
Fair enough. I didn't realize from the diff that there was a new plugin. That was the missing link. :+1: